.github/workflows/sync_issue_to_jira.yaml

```
@@ -0,0 +1,21 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Sync issue to Jira
+
+on:
+  issues:
+    types: [opened, reopened, closed]
+
+jobs:
+  sync:
+    name: Sync GitHub issue to Jira
+    uses: canonical/data-platform-workflows/.github/workflows/sync_issue_to_jira.yaml@v2
```

This can be bumped I think?
```suggestion
    uses: canonical/data-platform-workflows/.github/workflows/sync_issue_to_jira.yaml@v11
```